ASICBlocks/LoadData_ePortRX.py

```diff
@@ -71,6 +76,7 @@ def loadEportRXData(_inputDir, simEnergy=False, alignmentTime=324):
     df.set_index(['Orbit','BX'], inplace=True)
 
     df_BX_CNT = pd.DataFrame(headers, columns=['BX_CNT'], index=df.index)
+    df_BX_CNT.iloc[0] = 31
```

Comment:
I believe this is going to give the wrong header value.  It would only have the BC0 header for the very first BX, without any check of whether or not that is a BX of 0

Reply:
Reverted.